docs/absolute.md

```diff
@@ -356,6 +356,55 @@ billion.toDateTime(utc).difference(epoch.toDateTime(utc), { largestUnit: 'years'
   // => P31Y8M8DT1H46M40S
 ```
 
+### absolute.**round**(_options_: object) : Temporal.Absolute
+
+**Parameters:**
+- `options` (object): An object with properties representing options for the operation.
+  The following options are recognized:
+  - `smallestUnit` (required string): The unit to round to.
+    Valid values are `'minute'`, `'second'`, `'millisecond'`, `'microsecond'`, and `'nanosecond'`.
```

Comment:
I've added a note "Hour increments are not allowed, since `Temporal.Absolute` has no time zone and therefore it is not defined where the starting point of the hour is."
Although on reflection that seems like a flimsy justification, either minutes should be disallowed for the same reason, or hours should be allowed with the same starting point as minutes.

Reply:
I will open a new issue about changing this to be more consistent. I'd rather not delay this PR on account of this.

Reply:
→ https://github.com/tc39/proposal-temporal/issues/908